src/models/civs/bonuses.ts

```diff
@@ -217,4 +219,120 @@ export const civsBonuses: CivilizationBonusesEntry[] = [
 			},
 		],
 	},
+
+	{
+		name: 'Burmese',
+		focuses: [Focus.Monk, Focus.Elephant],
+		bonuses: [
+			// Traits
+			{
+				team: true,
+				description: 'Relics are visible on the map from the game start',
+				focuses: [Focus.Monk, Focus.ResourceGold],
```

Comment:
We need to add `Focus.Relic` (c.f. Lithuanians, add to Aztecs), and I think `Focus.Monk` should be removed here. Relics involve monks but they don't particularly benefit each other (all civs have access to monks and they aren't a burden to build IOT gather relics).